### ISSUE TITLE
Adding specific books results in blank / white page and console error

### ISSUE THREAD

**Issue by delize** [Readarr/Readarr#1906]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

1. Go to readarr/add/search?term=Star%20Trek%20Cookbook
2. Add any of the specific books listed (not the authors)
3. Click `Add $Book`
4. Blanks to white page
5. Go "back" a page, still have white page
6. Hard Refresh, and Readarr loads

### Expected Behavior

After step 3 in the current behavior, it should simply go back to the search results.

### Environment

```markdown
- OS: Ubuntu base
- Readarr: Readarr 0.1.1.1426 / Package Version 0.1.1.1426-ls137 by linuxserver.io
- Docker Install: Yes
- Using Reverse Proxy: Yes, SWAG
- Browser: Chrome 106
- Database: Sqlite 3.36.0
```

### What branch are you running?

Nightly

### Trace Logs?

```
instrument.js:109 TypeError: Cannot read properties of undefined (reading 'find')
    at AddNewItem.js:151:53
    at Array.map (<anonymous>)
    at o.render (AddNewItem.js:140:25)
    at Wa (react-dom.production.min.js:187:188)
    at Fa (react-dom.production.min.js:186:173)
    at Ws (react-dom.production.min.js:269:427)
    at Mu (react-dom.production.min.js:250:347)
    at wu (react-dom.production.min.js:250:278)
    at ku (react-dom.production.min.js:250:138)
    at pu (react-dom.production.min.js:243:163)
    at react-dom.production.min.js:123:115
    at t.unstable_runWithPriority (scheduler.production.min.js:18:343)
    at Go (react-dom.production.min.js:122:325)
    at Vo (react-dom.production.min.js:123:61)
    at Uo (react-dom.production.min.js:122:428)
    at hu (react-dom.production.min.js:244:190)
    at Object.notify (Subscription.js:19:7)
    at t.notifyNestedSubs (Subscription.js:90:20)
    at t.handleChangeWrapper (Subscription.js:95:12)
    at y (redux.js:296:32)
    at index.js:11:16
    at middleware.js:26:18
    at createSentryMiddleware.js:74:14
    at dispatch (redux.js:659:28)
    at Object.<anonymous> (searchActions.js:178:7)
    at f (callbacks.js:84:31)
    at fireWith (callbacks.js:214:7)
    at f (deferred.js:183:33)
    at p (deferred.js:192:12)
    at o (helpers.js:73:23)
```

```


(anonymous) | @ | instrument.js:109
-- | -- | --
  | os | @ | react-dom.production.min.js:216
  | n.callback | @ | react-dom.production.min.js:216
  | di | @ | react-dom.production.min.js:131
  | ds | @ | react-dom.production.min.js:220
  | ju | @ | react-dom.production.min.js:259
  | t.unstable_runWithPriority | @ | scheduler.production.min.js:18
  | Go | @ | react-dom.production.min.js:122
  | Lu | @ | react-dom.production.min.js:252
  | pu | @ | react-dom.production.min.js:243
  | (anonymous) | @ | react-dom.production.min.js:123
  | t.unstable_runWithPriority | @ | scheduler.production.min.js:18
  | Go | @ | react-dom.production.min.js:122
  | Vo | @ | react-dom.production.min.js:123
  | Uo | @ | react-dom.production.min.js:122
  | hu | @ | react-dom.production.min.js:244
  | notify | @ | Subscription.js:19
  | t.notifyNestedSubs | @ | Subscription.js:90
  | t.handleChangeWrapper | @ | Subscription.js:95
  | y | @ | redux.js:296
  | (anonymous) | @ | index.js:11
  | (anonymous) | @ | middleware.js:26
  | (anonymous) | @ | createSentryMiddleware.js:74
  | dispatch | @ | redux.js:659
  | (anonymous) | @ | searchActions.js:178
  | f | @ | callbacks.js:84
  | fireWith | @ | callbacks.js:214
  | f | @ | deferred.js:183
  | p | @ | deferred.js:192
  | o | @ | helpers.js:73
  | setTimeout (async) |   |  
  | (anonymous) | @ | trycatch.js:94
  | (anonymous) | @ | deferred.js:230
  | f | @ | callbacks.js:84
  | fireWith | @ | callbacks.js:214
  | fire | @ | callbacks.js:222
  | f | @ | callbacks.js:84
  | fireWith | @ | callbacks.js:214
  | R | @ | ajax.js:806
  | (anonymous) | @ | xhr.js:99
  | o | @ | helpers.js:73
  | load (async) |   |  
  | u | @ | object.js:35
  | (anonymous) | @ | trycatch.js:129
  | (anonymous) | @ | trycatch.js:126
  | (anonymous) | @ | instrument.js:248
  | send | @ | xhr.js:151
  | ajax | @ | ajax.js:700
  | $ | @ | createAjaxRequest.js:48
  | search/addBook | @ | searchActions.js:168
  | (anonymous) | @ | thunks.js:13
  | (anonymous) | @ | index.js:8
  | (anonymous) | @ | middleware.js:26
  | (anonymous) | @ | createSentryMiddleware.js:74
  | n.<computed> | @ | bindActionCreators.js:9
  | t.onAddBookPress | @ | AddNewBookModalContentConnector.js:67
  | r.onAddBookPress | @ | AddNewBookModalContent.js:37
  | onClick | @ | Link.js:19
  | We | @ | react-dom.production.min.js:52
  | $e | @ | react-dom.production.min.js:52
  | (anonymous) | @ | react-dom.production.min.js:53
  | Mr | @ | react-dom.production.min.js:100
  | Tr | @ | react-dom.production.min.js:101
  | (anonymous) | @ | react-dom.production.min.js:113
  | Ye | @ | react-dom.production.min.js:292
  | (anonymous) | @ | react-dom.production.min.js:50
  | Cr | @ | react-dom.production.min.js:105
  | Jt | @ | react-dom.production.min.js:75
  | Xt | @ | react-dom.production.min.js:74
  | t.unstable_runWithPriority | @ | scheduler.production.min.js:18
  | Go | @ | react-dom.production.min.js:122
  | Pe | @ | react-dom.production.min.js:292
  | Zt | @ | react-dom.production.min.js:73
  | o | @ | helpers.js:73

```

```2022-10-24 11:31:59.9|Trace|CommandExecutor|RefreshAuthorCommand -> RefreshAuthorService
2022-10-24 11:31:59.9|Trace|CommandQueueManager|Marking command as started: RefreshAuthor
2022-10-24 11:31:59.9|Trace|EventAggregator|Publishing CommandUpdatedEvent
2022-10-24 11:31:59.9|Trace|EventAggregator|CommandUpdatedEvent -> CommandController
2022-10-24 11:31:59.9|Trace|EventAggregator|CommandUpdatedEvent <- CommandController
2022-10-24 11:31:59.9|Trace|Http|Res: 492 [POST] /readarr/api/v1/book: 201.Created (2070 ms)
2022-10-24 11:31:59.9|Debug|BookInfoProxy|Getting Author details GoodreadsId of 5415687
2022-10-24 11:31:59.9|Info|RefreshAuthorService|Updating Info for Chelsea Monroe-Cassel
2022-10-24 11:31:59.9|Trace|EventAggregator|Publishing CommandUpdatedEvent
2022-10-24 11:31:59.9|Trace|EventAggregator|CommandUpdatedEvent -> CommandController
2022-10-24 11:31:59.9|Trace|EventAggregator|CommandUpdatedEvent <- CommandController
2022-10-24 11:31:59.9|Debug|AuthorMetadataRepository|1 author metadata up to date; Updating 0, Adding 0 author metadata entries.
2022-10-24 11:31:59.9|Trace|RefreshAuthorService|local: 39 remote: 39
2022-10-24 11:31:59.9|Trace|RefreshAuthorService|Updating Author [5415687][Chelsea Monroe-Cassel]
2022-10-24 11:31:59.9|Trace|RefreshAuthorService|updated: False forceUpdateFileTags: False
2022-10-24 11:31:59.9|Trace|MetadataProfileService|Filtering:
[49695188][World of Warcraft: The Official Cookbook]
[68194584][Overwatch: The Official Cookbook]
[56767249][Hearthstone: Innkeeper's Tavern Cookbook]
[49680837][The Shire Cookbook]
[62941721][The Elder Scrolls: The Official Cookbook]
[18537276][A Feast of Ice and Fire: The Official Game of Thrones Companion Cookbook]
[69431723][Firefly: The Big Damn Cookbook]
[68194031][Star Wars: Galaxy's Edge: The Official Black Spire Outpost Cookbook]
[43446298][From the Sands of Dorne: A Feast of Ice & Fire Companion Cookbook]
[88291018][The elder scrolls -livre de cuisine off.]
[91709231][The Star Trek Cookbook]
[94659569][World of Warcraft - Nouvelles saveurs d'Azeroth : Le livre de cuisine officiel]
[84736548][World of Warcraft: The Official Cookbook Gift Set]
[95882882][World of Warcraft: Das offizielle Kochbuch-Geschenkset: Kochbuch mit beidseitiger Horde- und Allianz-SchÃ¼rze]
[85028344][The Elder ScrollsÂ®: The Official Cookbook Gift Set:]
[91345237][World of Warcraft: New Flavors of Azeroth Gift Set Edition]
[95359640][World of Warcraft: New Flavors of Azeroth - The Official Cookbook]
[86883705][World of Warcraft: New Flavors of Azeroth: The Official Cookbook]
[91345239][Star Wars: Galaxy's Edge Gift Set Edition: The Official Black Spire Outpost Cookbook]
2022-10-24 11:31:59.9|Trace|MetadataProfileService|Skipping 14 Book because rating criteria not met:
[68194584][Overwatch: The Official Cookbook]
[56767249][Hearthstone: Innkeeper's Tavern Cookbook]
[49680837][The Shire Cookbook]
[69431723][Firefly: The Big Damn Cookbook]
[43446298][From the Sands of Dorne: A Feast of Ice & Fire Companion Cookbook]
[88291018][The elder scrolls -livre de cuisine off.]
[94659569][World of Warcraft - Nouvelles saveurs d'Azeroth : Le livre de cuisine officiel]
[84736548][World of Warcraft: The Official Cookbook Gift Set]
[95882882][World of Warcraft: Das offizielle Kochbuch-Geschenkset: Kochbuch mit beidseitiger Horde- und Allianz-SchÃ¼rze]
[85028344][The Elder ScrollsÂ®: The Official Cookbook Gift Set:]
[91345237][World of Warcraft: New Flavors of Azeroth Gift Set Edition]
[95359640][World of Warcraft: New Flavors of Azeroth - The Official Cookbook]
[86883705][World of Warcraft: New Flavors of Azeroth: The Official Cookbook]
[91345239][Star Wars: Galaxy's Edge Gift Set Edition: The Official Black Spire Outpost Cookbook]
2022-10-24 11:31:59.9|Trace|MetadataProfileService|Skipping 3 Edition because edition language not allowed:
[36611097][World of Warcraft: OficiÃ¡lnÃ­ kuchaÅ™ka]
[35272039][World of Warcraft: Das offizielle Kochbuch]
[56477757][ÐžÑ„Ð¸Ñ†Ð¸Ð°Ð»ÑŒÐ½Ð°Ñ Ð¿Ð¾Ð²Ð°Ñ€ÐµÐ½Ð½Ð°Ñ ÐºÐ½Ð¸Ð³Ð° World of Warcraft]
2022-10-24 11:31:59.9|Trace|MetadataProfileService|Skipping 1 Edition because edition language not allowed:
[59554317][The Elder Scrolls. ÐžÑ„Ð¸Ñ†Ð¸Ð°Ð»ÑŒÐ½Ñ‹Ð¹ ÑÐ±Ð¾Ñ€Ð½Ð¸Ðº Ñ€ÐµÑ†ÐµÐ¿Ñ‚Ð¾Ð²]
2022-10-24 11:31:59.9|Trace|MetadataProfileService|Skipping 7 Edition because edition language not allowed:
[18004165][A Game of Thrones â€“ Das offizielle Kochbuch]
[41556352][Game of Thrones, Le TrÃ´ne de Fer : le livre des festins]
[28480751][Uczta lodu i ognia]
[17436235][FestÃ­n de Hielo y Fuego: Libro oficial de cocina de Juego de tronos]
[17886282][FestÃ­n de Hielo y Fuego]
[23595979][ÐŸÐ¸Ñ€ Ð›ÑŒÐ´Ð° Ð¸ ÐžÐ³Ð½Ñ. ÐžÑ„Ð¸Ñ†Ð¸Ð°Ð»ÑŒÐ½Ð°Ñ Ð¿Ð¾Ð²Ð°Ñ€ÐµÐ½Ð½Ð°Ñ ÐºÐ½Ð¸Ð³Ð° "Ð˜Ð³Ñ€Ñ‹ Ð¿Ñ€ÐµÑÑ‚Ð¾Ð»Ð¾Ð²"]
[30828960][Hostina ledu a ohnÄ›: OficiÃ¡lnÃ­ kuchaÅ™ka Game of Thrones]
2022-10-24 11:31:59.9|Debug|RefreshAuthorService|[5415687][Chelsea Monroe-Cassel] 1 books up to date. Adding 4, Updating 0, Merging 0, Deleting 0.
2022-10-24 11:31:59.9|Info|RefreshBookService|Updating Info for The Star Trek Cookbook
2022-10-24 11:31:59.9|Trace|EventAggregator|Publishing CommandUpdatedEvent
2022-10-24 11:31:59.9|Trace|EventAggregator|CommandUpdatedEvent -> CommandController
2022-10-24 11:31:59.9|Trace|EventAggregator|CommandUpdatedEvent <- CommandController
2022-10-24 11:31:59.9|Trace|RefreshBookService|Ensuring parent author exists [5415687]
2022-10-24 11:31:59.9|Trace|RefreshBookService|Updating Book [91709231][The Star Trek Cookbook]
2022-10-24 11:31:59.9|Trace|RefreshBookService|updated: False forceUpdateFileTags: False
2022-10-24 11:31:59.9|Debug|RefreshBookService|[91709231][The Star Trek Cookbook] 3 editions up to date. Adding 0, Updating 0, Merging 0, Deleting 0.
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [58438643][The Star Trek Cookbook] monitored: True
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [60461230][The Star Trek Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [58439619][The Star Trek Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|ConfigService|Using default config value for 'writeaudiotags' defaultValue:'No'
2022-10-24 11:31:59.9|Trace|ConfigService|Using default config value for 'writebooktags' defaultValue:'NewFiles'
2022-10-24 11:31:59.9|Trace|RefreshBookService|Saving Book [91709231][The Star Trek Cookbook]
2022-10-24 11:31:59.9|Debug|RefreshBookService|Finished Book refresh for [91709231][The Star Trek Cookbook]
2022-10-24 11:31:59.9|Info|RefreshBookService|Updating Info for World of Warcraft: The Official Cookbook
2022-10-24 11:31:59.9|Trace|EventAggregator|Publishing CommandUpdatedEvent
2022-10-24 11:31:59.9|Trace|EventAggregator|CommandUpdatedEvent -> CommandController
2022-10-24 11:31:59.9|Trace|EventAggregator|CommandUpdatedEvent <- CommandController
2022-10-24 11:31:59.9|Trace|RefreshBookService|Ensuring parent author exists [5415687]
2022-10-24 11:31:59.9|Trace|RefreshBookService|Updating Book [49695188][World of Warcraft: The Official Cookbook]
2022-10-24 11:31:59.9|Trace|RefreshBookService|updated: False forceUpdateFileTags: False
2022-10-24 11:31:59.9|Debug|RefreshBookService|[49695188][World of Warcraft: The Official Cookbook] 0 editions up to date. Adding 6, Updating 0, Merging 0, Deleting 0.
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [33005059][World of Warcraft: The Official Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [33005057][World of Warcraft: The Official Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [32294996][World of Warcraft the Official Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [33124526][World of Warcraft: The Official Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [46134293][CHELSEA MONROE-CASSEL - WORLD] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [29430594][World of Warcraft: The Official Cookbook] monitored: True
2022-10-24 11:31:59.9|Trace|ConfigService|Using default config value for 'writeaudiotags' defaultValue:'No'
2022-10-24 11:31:59.9|Trace|ConfigService|Using default config value for 'writebooktags' defaultValue:'NewFiles'
2022-10-24 11:31:59.9|Trace|RefreshBookService|Saving Book [49695188][World of Warcraft: The Official Cookbook]
2022-10-24 11:31:59.9|Trace|EventAggregator|Publishing BookUpdatedEvent
2022-10-24 11:31:59.9|Trace|EventAggregator|BookUpdatedEvent -> AuthorStatisticsService
2022-10-24 11:31:59.9|Trace|EventAggregator|BookUpdatedEvent <- AuthorStatisticsService
2022-10-24 11:31:59.9|Trace|EventAggregator|BookUpdatedEvent -> BookController
2022-10-24 11:31:59.9|Trace|EventAggregator|BookUpdatedEvent <- BookController
2022-10-24 11:31:59.9|Debug|RefreshBookService|Finished Book refresh for [49695188][World of Warcraft: The Official Cookbook]
2022-10-24 11:31:59.9|Info|RefreshBookService|Updating Info for The Elder Scrolls: The Official Cookbook
2022-10-24 11:31:59.9|Trace|EventAggregator|Publishing CommandUpdatedEvent
2022-10-24 11:31:59.9|Trace|EventAggregator|CommandUpdatedEvent -> CommandController
2022-10-24 11:31:59.9|Trace|EventAggregator|CommandUpdatedEvent <- CommandController
2022-10-24 11:31:59.9|Trace|RefreshBookService|Ensuring parent author exists [5415687]
2022-10-24 11:31:59.9|Trace|RefreshBookService|Updating Book [62941721][The Elder Scrolls: The Official Cookbook]
2022-10-24 11:31:59.9|Trace|RefreshBookService|updated: False forceUpdateFileTags: False
2022-10-24 11:31:59.9|Debug|RefreshBookService|[62941721][The Elder Scrolls: The Official Cookbook] 0 editions up to date. Adding 9, Updating 0, Merging 0, Deleting 0.
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [43611898][The Elder Scrolls: The Official Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [40539188][The Elder Scrolls: The Official Cookbook] monitored: True
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [58508933][The Elder Scrolls: The Official Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [48946213][The Elder Scrolls: The Official Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [56414297][The Elder Scrolls: The Official Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [49631994][The Elder Scrolls: Das offizielle Kochbuch: Rezepte aus Himmelsrand, Morrowind und ganz Tamriel] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [62612286][The Elder Scrolls: The Official Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [58486564][The Elder Scrolls: The Official Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [61057704][The Elder Scrolls: The Official Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|ConfigService|Using default config value for 'writeaudiotags' defaultValue:'No'
2022-10-24 11:31:59.9|Trace|ConfigService|Using default config value for 'writebooktags' defaultValue:'NewFiles'
2022-10-24 11:31:59.9|Trace|RefreshBookService|Saving Book [62941721][The Elder Scrolls: The Official Cookbook]
2022-10-24 11:31:59.9|Trace|EventAggregator|Publishing BookUpdatedEvent
2022-10-24 11:31:59.9|Trace|EventAggregator|BookUpdatedEvent -> AuthorStatisticsService
2022-10-24 11:31:59.9|Trace|EventAggregator|BookUpdatedEvent <- AuthorStatisticsService
2022-10-24 11:31:59.9|Trace|EventAggregator|BookUpdatedEvent -> BookController
2022-10-24 11:31:59.9|Trace|EventAggregator|BookUpdatedEvent <- BookController
2022-10-24 11:31:59.9|Debug|RefreshBookService|Finished Book refresh for [62941721][The Elder Scrolls: The Official Cookbook]
2022-10-24 11:31:59.9|Info|RefreshBookService|Updating Info for A Feast of Ice and Fire: The Official Game of Thrones Companion Cookbook
2022-10-24 11:31:59.9|Trace|EventAggregator|Publishing CommandUpdatedEvent
2022-10-24 11:31:59.9|Trace|EventAggregator|CommandUpdatedEvent -> CommandController
2022-10-24 11:31:59.9|Trace|EventAggregator|CommandUpdatedEvent <- CommandController
2022-10-24 11:31:59.9|Trace|RefreshBookService|Ensuring parent author exists [5415687]
2022-10-24 11:31:59.9|Trace|RefreshBookService|Updating Book [18537276][A Feast of Ice and Fire: The Official Game of Thrones Companion Cookbook]
2022-10-24 11:31:59.9|Trace|RefreshBookService|updated: False forceUpdateFileTags: False
2022-10-24 11:31:59.9|Debug|RefreshBookService|[18537276][A Feast of Ice and Fire: The Official Game of Thrones Companion Cookbook] 0 editions up to date. Adding 6, Updating 0, Merging 0, Deleting 0.
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [40974456][Banquete de Gelo & Fogo. O Livro Oficial de Receitas de Game of Thrones] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [18415770][Feast of Ice and Fire: The Official Game of Thrones Companion Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [17678490][A Feast of Ice and Fire: The Official Game of Thrones Companion Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [13531258][A Feast of Ice and Fire: The Official Game of Thrones Companion Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [18701126][A Feast of Ice and Fire: The Official Game of Thrones Companion Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [13329738][A Feast of Ice and Fire: The Official Companion Cookbook] monitored: True
2022-10-24 11:31:59.9|Trace|ConfigService|Using default config value for 'writeaudiotags' defaultValue:'No'
2022-10-24 11:31:59.9|Trace|ConfigService|Using default config value for 'writebooktags' defaultValue:'NewFiles'
2022-10-24 11:31:59.9|Trace|RefreshBookService|Saving Book [18537276][A Feast of Ice and Fire: The Official Game of Thrones Companion Cookbook]
2022-10-24 11:31:59.9|Trace|EventAggregator|Publishing BookUpdatedEvent
2022-10-24 11:31:59.9|Trace|EventAggregator|BookUpdatedEvent -> AuthorStatisticsService
2022-10-24 11:31:59.9|Trace|EventAggregator|BookUpdatedEvent <- AuthorStatisticsService
2022-10-24 11:31:59.9|Trace|EventAggregator|BookUpdatedEvent -> BookController
2022-10-24 11:31:59.9|Trace|Http|Res: 168 [GET] /readarr/signalr/messages?access_token=(removed)&id=CZo4lm4dUmlldbukUUAcmQ: 101.SwitchingProtocols (5636487 ms)
2022-10-24 11:31:59.9|Trace|EventAggregator|BookUpdatedEvent <- BookController
2022-10-24 11:31:59.9|Debug|RefreshBookService|Finished Book refresh for [18537276][A Feast of Ice and Fire: The Official Game of Thrones Companion Cookbook]
2022-10-24 11:31:59.9|Info|RefreshBookService|Updating Info for Star Wars: Galaxy's Edge: The Official Black Spire Outpost Cookbook
2022-10-24 11:31:59.9|Trace|EventAggregator|Publishing CommandUpdatedEvent
2022-10-24 11:31:59.9|Trace|EventAggregator|CommandUpdatedEvent -> CommandController
2022-10-24 11:31:59.9|Trace|EventAggregator|CommandUpdatedEvent <- CommandController
2022-10-24 11:31:59.9|Trace|RefreshBookService|Ensuring parent author exists [5415687]
2022-10-24 11:31:59.9|Trace|RefreshBookService|Updating Book [68194031][Star Wars: Galaxy's Edge: The Official Black Spire Outpost Cookbook]
2022-10-24 11:31:59.9|Trace|RefreshBookService|updated: False forceUpdateFileTags: False
2022-10-24 11:31:59.9|Debug|RefreshBookService|[68194031][Star Wars: Galaxy's Edge: The Official Black Spire Outpost Cookbook] 0 editions up to date. Adding 2, Updating 0, Merging 0, Deleting 0.
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [43822113][Star Wars: Galaxy's Edge: The Official Black Spire Outpost Cookbook] monitored: True
2022-10-24 11:31:59.9|Trace|RefreshBookService|release: [49221450][Star Wars Galaxys Edge Cookbook] monitored: False
2022-10-24 11:31:59.9|Trace|ConfigService|Using default config value for 'writeaudiotags' defaultValue:'No'
2022-10-24 11:31:59.9|Trace|ConfigService|Using default config value for 'writebooktags' defaultValue:'NewFiles'
2022-10-24 11:31:59.9|Trace|RefreshBookService|Saving Book [68194031][Star Wars: Galaxy's Edge: The Official Black Spire Outpost Cookbook]
2022-10-24 11:31:59.9|Trace|EventAggregator|Publishing BookUpdatedEvent
2022-10-24 11:31:59.9|Trace|EventAggregator|BookUpdatedEvent -> AuthorStatisticsService
2022-10-24 11:31:59.9|Trace|EventAggregator|BookUpdatedEvent <- AuthorStatisticsService
2022-10-24 11:31:59.9|Trace|EventAggregator|BookUpdatedEvent -> BookController
2022-10-24 11:31:59.9|Trace|EventAggregator|BookUpdatedEvent <- BookController
2022-10-24 11:31:59.9|Debug|RefreshBookService|Finished Book refresh for [68194031][Star Wars: Galaxy's Edge: The Official Black Spire Outpost Cookbook]
2022-10-24 11:31:59.9|Trace|EventAggregator|Publishing BookInfoRefreshedEvent
2022-10-24 11:31:59.9|Trace|EventAggregator|BookInfoRefreshedEvent -> TrackedDownloadService
2022-10-24 11:31:59.9|Trace|EventAggregator|BookInfoRefreshedEvent <- TrackedDownloadService
2022-10-24 11:31:59.9|Trace|EventAggregator|BookInfoRefreshedEvent -> BookAddedService
2022-10-24 11:31:59.9|Trace|EventAggregator|BookInfoRefreshedEvent <- BookAddedService
2022-10-24 11:31:59.9|Trace|RefreshAuthorService|Saving Author [5415687][Chelsea Monroe-Cassel]
2022-10-24 11:32:00.0|Trace|EventAggregator|Publishing AuthorEditedEvent
2022-10-24 11:32:00.0|Trace|EventAggregator|AuthorEditedEvent -> AuthorEditedService
2022-10-24 11:32:00.0|Trace|EventAggregator|AuthorEditedEvent <- AuthorEditedService
2022-10-24 11:32:00.0|Trace|EventAggregator|AuthorEditedEvent -> AuthorController
2022-10-24 11:32:00.0|Trace|EventAggregator|AuthorEditedEvent <- AuthorController
2022-10-24 11:32:00.0|Trace|EventAggregator|Publishing AuthorUpdatedEvent
2022-10-24 11:32:00.0|Trace|EventAggregator|AuthorUpdatedEvent -> AuthorStatisticsService
2022-10-24 11:32:00.0|Trace|EventAggregator|AuthorUpdatedEvent <- AuthorStatisticsService
2022-10-24 11:32:00.0|Trace|EventAggregator|AuthorUpdatedEvent -> AuthorController
2022-10-24 11:32:00.0|Trace|EventAggregator|AuthorUpdatedEvent <- AuthorController
2022-10-24 11:32:00.0|Trace|EventAggregator|Publishing AuthorRefreshCompleteEvent
2022-10-24 11:32:00.0|Trace|EventAggregator|AuthorRefreshCompleteEvent ~> MediaCoverService
2022-10-24 11:32:00.0|Debug|RefreshAuthorService|Finished Author refresh for [5415687][Chelsea Monroe-Cassel]
2022-10-24 11:32:00.0|Trace|HttpClient|Req: [GET] https://images-na.ssl-images-amazon.com/images/S/compressed.photo.goodreads.com/authors/1331682090i/5415687.jpg
2022-10-24 11:32:00.0|Trace|ConfigService|Using default config value for 'rescanafterrefresh' defaultValue:'Always'
2022-10-24 11:32:00.0|Trace|CommandQueueManager|Publishing RescanFolders
2022-10-24 11:32:00.0|Trace|CommandQueueManager|Checking if command is queued or started: RescanFolders
2022-10-24 11:32:00.0|Trace|CommandQueueManager|Inserting new command: RescanFolders
2022-10-24 11:32:00.0|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2022-10-24 11:32:00.0|Trace|CommandQueueManager|Updating command status
2022-10-24 11:32:00.0|Trace|CommandExecutor|RescanFoldersCommand -> DiskScanService
2022-10-24 11:32:00.0|Trace|CommandQueueManager|Marking command as started: RescanFolders
2022-10-24 11:32:00.0|Trace|EventAggregator|Publishing CommandUpdatedEvent
2022-10-24 11:32:00.0|Trace|EventAggregator|CommandUpdatedEvent -> CommandController
2022-10-24 11:32:00.0|Trace|EventAggregator|CommandUpdatedEvent <- CommandController
2022-10-24 11:32:00.0|Trace|EventAggregator|Publishing CommandExecutedEvent
2022-10-24 11:32:00.0|Trace|EventAggregator|CommandExecutedEvent -> TaskManager
2022-10-24 11:32:00.0|Trace|EventAggregator|CommandExecutedEvent <- TaskManager
2022-10-24 11:32:00.0|Trace|EventAggregator|CommandExecutedEvent -> TaskController
2022-10-24 11:32:00.0|Trace|EventAggregator|CommandExecutedEvent <- TaskController
2022-10-24 11:32:00.0|Trace|CommandExecutor|RefreshAuthorCommand <- RefreshAuthorService [00:00:00.0970600]
2022-10-24 11:32:00.0|Trace|EventAggregator|Publishing CommandUpdatedEvent
2022-10-24 11:32:00.0|Trace|EventAggregator|CommandUpdatedEvent -> CommandController
2022-10-24 11:32:00.0|Trace|EventAggregator|CommandUpdatedEvent <- CommandController
2022-10-24 11:32:00.0|Info|DiskScanService|Scanning /pool/books/Calibre Library/
2022-10-24 11:32:00.0|Trace|EventAggregator|Publishing CommandUpdatedEvent
2022-10-24 11:32:00.0|Trace|EventAggregator|CommandUpdatedEvent -> CommandController
2022-10-24 11:32:00.0|Trace|EventAggregator|CommandUpdatedEvent <- CommandController
2022-10-24 11:32:00.0|Trace|DiskScanService|{
  "name": "Calibre Library",
  "path": "/pool/books/Calibre Library/",
  "defaultMetadataProfileId": 1,
  "defaultQualityProfileId": 1,
  "defaultMonitorOption": "all",
  "defaultNewItemMonitorOption": "all",
  "defaultTags": [],
  "isCalibreLibrary": true,
  "calibreSettings": {
    "host": "calibre",
    "port": 8081,
    "urlBase": "",
    "username": "(removed)",
    "password": "(removed)",
    "library": "Calibre_Library",
    "outputFormat": "epub",
    "outputProfile": 0,
    "useSsl": false
  },
  "accessible": false,
  "id": 1
}
2022-10-24 11:32:00.0|Info|DiskScanService|Getting book list from calibre for /pool/books/Calibre Library/
2022-10-24 11:32:00.0|Trace|HttpClient|Req: [GET] http://calibre:8081/ajax/category/616c6c626f6f6b73/Calibre_Library?num=750&offset=0
2022-10-24 11:32:00.0|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2022-10-24 11:32:00.0|Trace|HttpClient|Res: [GET] http://calibre:8081/ajax/category/616c6c626f6f6b73/Calibre_Library?num=750&offset=0: 200.OK (164 bytes) (4 ms)
2022-10-24 11:32:00.0|Trace|HttpClient|Response content (164 bytes): {"total_num": 0, "sort_order": "desc", "offset": 0, "num": 0, "sort": "timestamp", "base_url": "/ajax/books_in/616c6c626f6f6b73/30/Calibre_Library", "book_ids": []}
2022-10-24 11:32:00.0|Debug|DiskScanService|0 book files were found in /pool/books/Calibre Library/
2022-10-24 11:32:00.0|Warn|DiskScanService|Scan folder /pool/books/Calibre Library/ is empty.
2022-10-24 11:32:00.0|Trace|DiskScanService|Finished getting track files for:
/pool/books/Calibre Library/ [00:00:00.0144621]
2022-10-24 11:32:00.0|Debug|MediaFileService|Filtering 0 files for unchanged files
2022-10-24 11:32:00.0|Trace|MediaFileService|Got 0 existing files
2022-10-24 11:32:00.0|Debug|ImportDecisionMaker|Analyzing 0/0 files.
2022-10-24 11:32:00.0|Debug|IdentificationService|Starting book identification
2022-10-24 11:32:00.0|Info|TrackGroupingService|Grouping 0 tracks
2022-10-24 11:32:00.0|Trace|EventAggregator|Publishing CommandUpdatedEvent
2022-10-24 11:32:00.0|Trace|EventAggregator|CommandUpdatedEvent -> CommandController
2022-10-24 11:32:00.0|Trace|EventAggregator|CommandUpdatedEvent <- CommandController
2022-10-24 11:32:00.0|Trace|TrackGroupingService|Folders:

2022-10-24 11:32:00.0|Debug|IdentificationService|Sorted 0 tracks into 0 releases in 1ms
2022-10-24 11:32:00.0|Debug|IdentificationService|Track identification for 0 tracks took 1ms
2022-10-24 11:32:00.0|Debug|DiskScanService|Import decisions complete [00:00:00.0039034]
2022-10-24 11:32:00.0|Info|ImportApprovedBooks|Importing 0 files
2022-10-24 11:32:00.0|Trace|EventAggregator|Publishing CommandUpdatedEvent
2022-10-24 11:32:00.0|Trace|EventAggregator|CommandUpdatedEvent -> CommandController
2022-10-24 11:32:00.0|Trace|EventAggregator|CommandUpdatedEvent <- CommandController
2022-10-24 11:32:00.0|Debug|ImportApprovedBooks|Importing 0 files. replaceExisting: False
2022-10-24 11:32:00.0|Debug|ImportApprovedBooks|Inserted new trackfiles in 0ms
2022-10-24 11:32:00.0|Debug|DiskScanService|Inserted 0 new unmatched trackfiles
2022-10-24 11:32:00.0|Debug|DiskScanService|Updated info for 0 known files
2022-10-24 11:32:00.0|Info|DiskScanService|Completed scanning disk for Chelsea Monroe-Cassel
2022-10-24 11:32:00.0|Trace|EventAggregator|Publishing AuthorScannedEvent
2022-10-24 11:32:00.0|Trace|EventAggregator|AuthorScannedEvent -> UpdateBookFileService
2022-10-24 11:32:00.0|Trace|ConfigService|Using default config value for 'filedate' defaultValue:'None'
2022-10-24 11:32:00.0|Trace|EventAggregator|AuthorScannedEvent <- UpdateBookFileService
2022-10-24 11:32:00.0|Trace|EventAggregator|AuthorScannedEvent -> ExistingExtraFileService
2022-10-24 11:32:00.0|Trace|EventAggregator|AuthorScannedEvent <- ExistingExtraFileService
2022-10-24 11:32:00.0|Trace|EventAggregator|AuthorScannedEvent -> AuthorScannedHandler
2022-10-24 11:32:00.0|Info|AuthorScannedHandler|[Chelsea Monroe-Cassel] was recently added, performing post-add actions
2022-10-24 11:32:00.0|Debug|BookMonitoredService|[Chelsea Monroe-Cassel] Setting book monitored status.
2022-10-24 11:32:00.0|Trace|EventAggregator|Publishing BookEditedEvent
```

[AB#3964](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/3964)

**Comment by Qstick** [comment]
Thanks for the report!
